src/java.net.http/share/classes/java/net/http/HttpRequest.java

```diff
@@ -303,6 +303,48 @@ protected HttpRequest() {}
         return new HttpRequestBuilderImpl(uri);
     }
 
+    /**
+     * Creates a {@code Builder} seeded from a {@code HttpRequest}.
```

Comment:
We use *an* HttpXXX consistently elsewhere. Please do the same here.

Reply:
Typo corrected in commit https://github.com/openjdk/jdk/pull/1059/commits/7928083ae954c44b5357064b86af0e1a3f53588b